src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -645,6 +597,75 @@ public ShopifyCustomer updateCustomer(final ShopifyCustomerUpdateRequest shopify
 		return shopifyCustomerRootResponse.getCustomer();
 	}
 
+	public ShopifyCustomer getCustomerById(final String id) {
+		final Response response = get(getWebTarget().path(CUSTOMERS).path(id));
+		final ShopifyCustomerRoot shopifyCustomerRootResponse = response.readEntity(ShopifyCustomerRoot.class);
+		return shopifyCustomerRootResponse.getCustomer();
+	}
+
+	public List<ShopifyCustomer> getCustomers(Response response) {
```

Comment:
I think we can hide this from SDK users since they aren't dealing with raw responses. Mark this as private and move towards the bottom of file with the other private methods.